server/@types/remandAndSentencingApi/index.d.ts

```diff
@@ -911,6 +911,9 @@ export interface components {
         | 'FINE'
         | 'LEGACY'
         | 'NON_CUSTODIAL'
+      hint?: {
```

Comment:
this is an auto generated class using https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/main/package.json#L28 command, is there some API work to do here?

Reply:
Yes, I will spin this off into a new ticket